graphql/resolvers.go

```diff
@@ -2419,10 +2424,17 @@ func (r *taskResolver) Annotation(ctx context.Context, obj *restModel.APITask) (
 		return nil, nil
 	}
 	apiAnnotation := restModel.APITaskAnnotationBuildFromService(*annotation)
-
-	//todo: get jira ticket objects before returning
-
-	return apiAnnotation, nil
+	issues, err := restModel.GetJiraTickets(apiAnnotation.Issues)
+	apiAnnotation.Issues = issues
+	if err != nil {
+		catcher.Add(err)
```

Comment:
catcher.Add(nil) just adds nothing, so there's no need to check if err != nil first. Also, logically this should be above the apiAnnotations.Issues = issue line (same for the lower block)